lib/screens/quick_game_screen.dart

```diff
@@ -33,7 +31,6 @@ class _QuickGameScreenState extends State<QuickGameScreen> {
       body: Container( 
         width: deviceSize.width,
         child: Column(  
-          // mainAxisAlignment: MainAxisAlignment.spaceBetween,
           crossAxisAlignment: CrossAxisAlignment.center,
           children: <Widget>[
             SizedBox(height: 25),
```

Comment:
Did the SizedBox help resolve the issue with centering?

Reply:
This wasn't my issue, it was with a different area. The Sizedbox helps by basically adding padding between widgets.